src/libraries/System.Memory/src/System/Buffers/Text/Base64Decoder.cs

```diff
@@ -588,52 +611,76 @@ private static unsafe void Ssse3Decode(ref byte* srcBytes, ref byte* destBytes,
             do
             {
                 AssertRead<Vector128<sbyte>>(src, srcStart, sourceLength);
-                Vector128<sbyte> str = Sse2.LoadVector128(src).AsSByte();
+                Vector128<byte> str = Vector128.LoadUnsafe(ref *src);
 
                 // lookup
-                Vector128<sbyte> hiNibbles = Sse2.And(Sse2.ShiftRightLogical(str.AsInt32(), 4).AsSByte(), mask2F);
-                Vector128<sbyte> loNibbles = Sse2.And(str, mask2F);
-                Vector128<sbyte> hi = Ssse3.Shuffle(lutHi, hiNibbles);
-                Vector128<sbyte> lo = Ssse3.Shuffle(lutLo, loNibbles);
+                Vector128<byte> hiNibbles = Vector128.ShiftRightLogical(str.AsInt32(), 4).AsByte();
```

Comment:
I should open an issue for `operator <<`, `operator >>`, and `operator >>>`.

We can then have this be just `(str.AsInt32() >>> 4).AsByte()`